borg/remote.py

```diff
@@ -166,7 +166,7 @@ def __init__(self, location, create=False, exclusive=False, lock_wait=None, lock
         self.preload_ids = []
         self.msgid = 0
         self.to_send = b''
-        self.cache = {}
+        self.get_cache = {}
```

Comment:
ok, it’s chunkid_to_msgids and pop_preload_msgid now.

Also moved the whole thing to be based on master